src/lib/util/smallUtils.ts

```diff
@@ -70,6 +70,12 @@ export function pluraliseItemName(name: string): string {
 	return name + (name.endsWith('s') ? '' : 's');
 }
 
+export function pluraliseItemNameWithQuantity(name: string, quantity: number): string {
+	const result = `${quantity} ${name}`;
+	if (quantity === 1) return result;
+	return pluraliseItemName(result);
+}
```

Comment:
This function seems odd to me, if its only 1, it will include the quantity, but if its anything more than 1, it wont show the quantity? I generally prefer now to just write: `1x Coal, 1x Egg, 5x Trout`, this way theres no pluralization needed.

Reply:
I was hoping that the unit tests written for this method would make its functionality clear, but I can try to elaborate: Even if `quantity != 1`, it will still be shown. Consider `name` is `"pike"` and `quantity` is `5`.
1. `result` === `"5 pike"`
2. `pluraliseItemName("5 pike")` is called
3. return "5 pike" + "s" => `"5 pikes"`

I added this method because I noticed the `1x item` format wasn't consistently used. Few examples I have on hand are the output of
- `k` (`Your minion finished killing 553 Man. Your Man KC is now 553.`),
- `mine` (`Your minion finished mining 327 Pure essence. You received 1,635 :mining: XP. (3k/Hr)`) and
- `fish` (`Your minion finished fishing 24 Sardine. You received 480 :fishing: XP. (18k/Hr)`)

It would facilitate correct English if someone wants to run those outputs through `pluraliseItemNameWithQuanity`, as a future refactoring.

I can change the output of `fish` to not use this function. Though, after reading this, do you think this function is still useful? Or should I remove it?

Reply:
I think using the 1x format is better and we should stick with that

Reply:
And the function itself? Delete?